.github/workflows/checks.yml

```diff
@@ -1,21 +1,21 @@
 name: Check Build
 
-on: [push, pull_request]
+on: [pull_request]
 
 jobs:
   build:
 
     strategy:
       matrix:
-        php: ["7.4"]
+        php: ["7.4", "8.0"]
+        include:
+          - php: "8.0"
+            composer_args: "--ignore-platform-reqs"
```

Comment:
Why do we need this? I don't get any errors when I `composer update` on PHP 8.

Reply:
true. I think when I created the PR some dev-dependency (or a dependency of one of them) didn't have PHP8 support.